### PR TITLE
Adobe Document Services PDF Services SDK 3.4.0 Release

### PR DESCRIPTION
**Pull request by varkumaradobe**
This PR aims at the following:

1. Added new samples to illustrate how to apply electronic seal over PDF documents.
2. Updated samples to illustrate the use OAuth based server to server credentials
3. Updated the samples to use the latest Adobe PDF Services SDK version 3.4.0.

### CODE REVIEW

File: src/main/java/com/adobe/pdfservices/operation/samples/combinepdf/CombinePDFWithPageRanges.java [hunk 1]
```diff
@@ -105,4 +106,5 @@ public static String createOutputFilePath(){
         return("output/CombinePDFWithPageRanges/combine" + timeStamp + ".pdf");
     }
 
+
```
Comment (vardhi) [inline src/main/java/com/adobe/pdfservices/operation/samples/combinepdf/CombinePDFWithPageRanges.java:109]: remove extra new line.

File: src/main/java/com/adobe/pdfservices/operation/samples/createpdf/CreatePDFFromHTMLWithInlineCSS.java [hunk 2]
```diff
@@ -20,6 +20,7 @@
 import com.adobe.pdfservices.operation.pdfops.CreatePDFOperation;
 import com.adobe.pdfservices.operation.pdfops.options.createpdf.CreatePDFOptions;
 import com.adobe.pdfservices.operation.pdfops.options.createpdf.PageLayout;
+import org.json.JSONObject;
```
Comment (vardhi) [inline src/main/java/com/adobe/pdfservices/operation/samples/createpdf/CreatePDFFromHTMLWithInlineCSS.java:23]: remove unused import

File: src/main/java/com/adobe/pdfservices/operation/samples/exportpdftoimages/ExportPDFToJPEG.java [hunk 3]
```diff
@@ -53,6 +55,7 @@ public static void main(String[] args) {
             FileRef sourceFileRef = FileRef.createFromLocalFile("src/main/resources/exportPDFToImageInput.pdf");
             exportPDFToImagesOperation.setInput(sourceFileRef);
 
+
```
Comment (vardhi) [inline src/main/java/com/adobe/pdfservices/operation/samples/exportpdftoimages/ExportPDFToJPEG.java:58]: remove extra line

**Review by vardhi — CHANGES_REQUESTED** [review]

File: src/main/java/com/adobe/pdfservices/operation/samples/combinepdf/CombinePDFWithPageRanges.java (hunk 1 above)
Comment (varkumaradobe) [reply src/main/java/com/adobe/pdfservices/operation/samples/combinepdf/CombinePDFWithPageRanges.java]: Done.

File: src/main/java/com/adobe/pdfservices/operation/samples/createpdf/CreatePDFFromHTMLWithInlineCSS.java (hunk 2 above)
Comment (varkumaradobe) [reply src/main/java/com/adobe/pdfservices/operation/samples/createpdf/CreatePDFFromHTMLWithInlineCSS.java]: Done

File: src/main/java/com/adobe/pdfservices/operation/samples/exportpdftoimages/ExportPDFToJPEG.java (hunk 3 above)
Comment (varkumaradobe) [reply src/main/java/com/adobe/pdfservices/operation/samples/exportpdftoimages/ExportPDFToJPEG.java]: Done

**Review by vardhi — APPROVED** [review]

**Pull request merged**